src/artm/messages.proto

```diff
@@ -502,11 +502,21 @@ message CollectionParserConfig {
   optional string docword_file_path = 2;
   optional string vocab_file_path = 3;
   optional string target_folder = 4;
-  optional int32 num_items_per_batch = 5 [default = 1000];
```

Comment:
return back old fields and add your ones under them (for backward compability we don't remove any fields from messages and don't change their position)